### PR TITLE
Mention line lengths in README

### PR DESCRIPTION
**Pull request by marioortizmanero**
## Description

This should fix what we discussed at #167 regarding line lengths that aren't checked automatically by `cargo fmt`. Let me know if you think it could be worded differently.

### CODE REVIEW

**Comment by ramsayleung** [comment]
It's a great point to point out the line lengths in `README.md`, but I am afraid of that other developers will miss the code guide part if they don't read the `README.md` carefully.

IMHO, it would be better to have a way to check line lengths automatically.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Please see the [changelog](./CHANGELOG.md) for a release history and indications
 
 If you find any problems or have suggestions about this crate, please submit an issue. Moreover, any pull request, code review and feedback are welcome.
 
+We use GitHub Actions to make sure the codebase is consistent (`cargo fmt`) and continuously tested (`cargo test`). We try to keep comments at a maximum of 80 characters of length (which isn't automatically checked by `cargo fmt`) and code at 120.
+
```
Comment (ramsayleung) [inline README.md:21]: Perhaps, we could have an `Code Guide` section?

**Comment by marioortizmanero** [comment]
> IMHO, it would be better to have a way to check line lengths automatically.

As I pointed out in a different PR, this wouldn't be possible:

> I think a script isn't a viable solution because some lines need to be longer than 80 characters when there's a long URL and such.

File: README.md (hunk 1 above)
Comment (marioortizmanero) [reply README.md:21]: done

**Review by ramsayleung — APPROVED** [review]

**Pull request merged**